scripts/docgen/main.ts

```diff
@@ -2,25 +2,33 @@ import { stdout } from "node:process"
 import { fromPackageRoot } from "@re-/node"
 import { DocGenConfig } from "./config.js"
 import { extractRepo } from "./extract.js"
+import { createWriteFilesConsumer } from "./snippets/writeFilesConsumer.js"
 import { writeRepo } from "./write.js"
 
-const fromRedoDevDocsDir = (...segments: string[]) =>
-    fromPackageRoot("redo.dev", "docs", ...segments)
+const fromRedoDevDir = (...segments: string[]) =>
+    fromPackageRoot("redo.dev", ...segments)
 
 export const config: DocGenConfig = {
     packages: [
         {
             path: "@re-/model",
             api: {
-                outDir: fromRedoDevDocsDir("model", "api")
+                outDir: fromRedoDevDir("docs", "model", "api")
             },
             snippets: {
                 sources: [
                     {
                         path: "docs/snippets/"
                     }
                 ],
-                targets: ["README.md"]
+                targets: ["README.md"],
+                consumers: [
+                    createWriteFilesConsumer({
+                        rootOutDir: fromRedoDevDir("static", "stackblitz"),
+                        transformFileName: (name) =>
+                            `${name.split("/").pop()}.raw`
```

Comment:
I like the approach here and it's exactly what I would do in similar cases, but for getting the last segment of a path you can use:

```ts
import { basename } from "node:path"

...
transformFileName: (path) => `${basename(path)}.raw`
...
```

It also occurs to me as I'm trying to figure out what's going on here that `transformFileName` should probably be called `transformRelativePath` (my bad).